config/set/php80.php

```diff
@@ -89,20 +85,6 @@
             'pg_result' => 'pg_fetch_result',
             'pg_setclientencoding' => 'pg_set_client_encoding',
         ]);
-    $rectorConfig->rule(OptionalParametersAfterRequiredRector::class);
```

Comment:
this is on php 8.0 https://php.watch/versions/8.0/deprecate-required-param-after-optional